assets/js/base/components/cart-checkout/totals/discount/index.js

```diff
@@ -92,7 +89,7 @@ const TotalsDiscount = ( {
 												'Remove coupon "%s"',
 												'woo-gutenberg-products-block'
 											),
-											filteredCouponCode
+											cartCoupon.text
```

Comment:
Shouldn't this be `cartCoupon.label` as well?

Reply:
Great spot! Updated